products/compute/api.yaml

```diff
@@ -11452,32 +11527,742 @@ objects:
+                            properties:
+                              - !ruby/object:Api::Type::NestedObject
+                                name: 'fixedDelay'
+                                description: |
+                                  Specifies the value of the fixed delay interval.
+                                properties:
+                                  - !ruby/object:Api::Type::Integer
+                                    name: 'nanos'
+                                    description: |
+                                      Span of time that's a fraction of a second at nanosecond resolution. Durations
+                                      less than one second are represented with a 0 `seconds` field and a positive
+                                      `nanos` field. Must be from 0 to 999,999,999 inclusive.
+                                  - !ruby/object:Api::Type::String
+                                    name: 'seconds'
+                                    required: true
+                                    description: |
+                                      Span of time at a resolution of a second. Must be from 0 to 315,576,000,000
+                                      inclusive.
+                              - !ruby/object:Api::Type::Double
+                                name: 'percentage'
+                                description: |
+                                  The percentage of traffic (connections/operations/requests) on which delay will
+                                  be introduced as part of fault injection. The value must be between 0.0 and
+                                  100.0 inclusive.
+                      - !ruby/object:Api::Type::NestedObject
+                        name: 'requestMirrorPolicy'
+                        description: |
+                          Specifies the policy on how requests intended for the route's backends are
+                          shadowed to a separate mirrored backend service. Loadbalancer does not wait for
+                          responses from the shadow service. Prior to sending traffic to the shadow
+                          service, the host / authority header is suffixed with -shadow.
+                        properties:
+                          - !ruby/object:Api::Type::ResourceRef
+                            name: 'backendService'
+                            resource: 'BackendService'
+                            imports: 'selfLink'
+                            description: |
+                              The BackendService resource being mirrored to.
+                      - !ruby/object:Api::Type::NestedObject
+                        name: 'retryPolicy'
+                        description: |
+                          Specifies the retry policy associated with this route.
+                        properties:
+                          - !ruby/object:Api::Type::Integer
+                            name: 'numRetries'
```

Comment:
Are this & the below value required? Do they take on defaults when not specified?

Reply:
I updated the docs for `perTryTimeout` and made `numRetries` required.